alert/doc/doc.md

```diff
@@ -1,5 +1,5 @@
 ```js script
-import { html, registerIconLibrary } from '~/md-layout';
+import { html, registerIconLibrary } from '@divriots/starter-kicks/md-layout';
```

Comment:
Why we should not use anymore the `~` ?

Reply:
@gqio https://github.com/divriots/webcomponents-editor/pull/2556

Reply:
I updated all paths in md/mdx/stories without more thoughts. Ideally should be replaced only when visible in the docs, but that's quite the burden to check everywhere whether it's visible or not.
(but for this starter it probably makes no sense actually)

Reply:
ok it's a major change this one I missed it

Reply:
actually for `md-layout` it doesn't make sense, I'd revert this change

otherwise LGTM!